doc/jsk_topic_tools/scripts/and_node.rst

```diff
@@ -0,0 +1,36 @@
+and_node.py
+===========
+
+
+What is this?
+-------------
+
+A node that takes bool values and returns the result of the ``and`` operation.
+
+
+Subscribing Topics
+------------------
+
+* ``~input01`` (``std_msgs/Bool``)
+
+  input bool value.
+
+* ``~input02`` (``std_msgs/Bool``)
+
+  input bool value.
+
+
+Publishing Topics
+-----------------
+
+* ``~output`` (``std_msgs/Bool``)
+
+  The result of the ``and`` operation.
+
+
+Parameters
+----------
+
+* ``~rate`` (Int, Default: ``100``)
+
+  Publishing rate [Hz].
```

Comment:
Could you add usege of the node with topic remapping?

```
rosrun jsk_topic_tools xxx input01:=yy
```

Or, I think creating launch is also useful. (both is OK)

```
roslaunch jsk_topic_tools xxx.launch input01:=yyy 
```

Reply:
Thanks. I added the sample launch program.
https://github.com/jsk-ros-pkg/jsk_common/blob/3fa6565dbc203546b8ac560243968dc46a6db606/doc/jsk_topic_tools/scripts/boolean_node.rst